pkg/docker/config/config.go

```diff
@@ -55,7 +56,27 @@ var (
 
 // SetAuthentication stores the username and password in the auth.json file
```

Comment:
Throughout, the documentation should be updated to say that the credentials are stored/read/…, but not refer to any specific files.

Reply:
Most of the functions still need updated documentation.